front/src/components/App/MaintenanceArea/AddAuction.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const AddAuction = (auctions: string[], newAuction: string) => {
```

Comment:
Zadanie nie dotyczyło aukcji tylko portalu aukcyjnego i raczej na pewno nie będzie to string tylko jakiś złożony typ. Dlatego warto stworzyć interfejs na którym oprzemy ten byt, żeby uniknąć primitive obsession który jest bardzo złą praktyką